lib/services/local/generic/generic_153.go

```diff
@@ -0,0 +1,105 @@
+/*
+ * Teleport
+ * Copyright (C) 2023  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package generic
+
+import (
+	"context"
+
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/lib/backend"
+	"github.com/gravitational/teleport/lib/services"
+)
+
+// NewService153 will return a new generic service for a given RFD 153-style resource.
+func NewService153[T types.ResourceMetadata](
```

Comment:
@Tener @codingllama I don't mean to helicopter in mid-review with naming nitpicks but I'm of a pretty strong opinion that we should not be encoding RFD numbers in type and file names. It is pretty confusing, if this is the recommended service interface each new resource needs to implement now let's just call it `Service` without any numbers. The comment can still reference relevant RFD.

Reply:
I believe Krzysztof is following suite from [Resource153](https://github.com/gravitational/teleport/blob/e0d345793f2b0202e69ea6440b584b2c0d86277f/api/types/resource_153.go#L42), which landed a while back.

No objections on renaming on my part, the problem is figuring out a good name for it.

Reply:
Yeah, a good name is a tricky part, and indeed I was following the naming convention set by `Resource153`. I'm open for suggestions for better names, though.

Reply:
`LegacyService` suggests there is another implementation around, which isn't true. The wrapper for new style resources makes use of the same `Service` under the hood.

How about renaming `Service153` to `ServiceWrapper`?